recipes/ucc/meta.yaml

```
@@ -0,0 +1,54 @@
+package:
+  name: ucc
+  version: v1.2.0
+
+source:
+  url: https://github.com/openucx/ucc/archive/refs/tags/v1.2.0.tar.gz
+  sha256: c1552797600835c0cf401b82dc89c4d27d5717f4fb805d41daca8e19f65e509d
+  patches:
+    - 0001-EC-CUDA-fix-build-with-12.2-and-older-arch-847.patch  # [(cuda_compiler_version or "None").startswith("12")]
+
+build:
+  skip: True  # [(not linux) or (cuda_compiler_version == 'None')]
```

```suggestion
  skip: True  # [not linux]
  skip: True  # [cuda_compiler_version == "None"]
  skip: True  # [cuda_compiler_version == "11.2"]
```
I feel like you want these skips? We dropped support for 11.2 a few days ago.

https://conda-forge.org/news/2024/03/06/dropping-cuda-112/